build.gradle

```diff
@@ -23,6 +23,8 @@ dependencies {
 	compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.9.2'
 	compile 'org.springframework.plugin:spring-plugin-core:1.2.0.RELEASE'
 	compile group: 'com.google.guava', name: 'guava', version: '27.0-jre'
+	runtimeOnly 'com.h2database:h2'
```

Comment:
I think you can use testImplementation instead of runtimeOnly: https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_configurations_graph